asn1ate/sema.py

```diff
@@ -217,7 +216,9 @@ def children(self):
         sema nodes.
         """
         # Collect all SemaNode members.
-        members = list(vars(self).values())
+        # We need to exclude the 'referenced_modules' member if it exists
```

Comment:
It'd be great to say something about why here. I'm guessing it is because `referenced_modules` is not a child in the 'normal' sense, but rather some kind of metadata.

Fold it into the "Collect all SemaNode members", as that's no longer what we're doing.